scripts/install-protoc.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Please check how I updated this file here:
https://github.com/networkservicemesh/networkservicemesh/pull/749/files

Reply:
Should be resolved now 👍